soh/soh/Enhancements/Autosave.cpp

```diff
@@ -0,0 +1,71 @@
+#include <libultraship/bridge.h>
+#include "AutoSave.h"
```

Comment:
```suggestion
#include "Autosave.h"
```

this is why the linux build is failing